build/devenv/environment.go

```diff
@@ -426,9 +476,105 @@ func NewEnvironment() (in *Cfg, err error) {
 	return in, Store(in)
 }
 
+// createJobs creates the jobs for the verifiers and executors on the CL nodes if they're in CL mode.
+func createJobs(in *Cfg, vIn []*services.VerifierInput, executorIn *services.ExecutorInput) error {
+	// Exit early, there are no nodes configured.
+	if len(in.NodeSets) == 0 {
+		return nil
+	}
+
+	clClients, err := clclient.New(in.NodeSets[0].Out.CLNodes)
+	if err != nil {
+		return fmt.Errorf("failed to connect CL node clients: %w", err)
+	}
+
+	roundRobin := NewRoundRobinAssignment(clClients)
+	for _, ver := range vIn {
+		switch ver.Mode {
+		case services.CL:
+			index, clClient := roundRobin.GetNext()
+
+			tomlConfig, err := ver.GenerateConfig()
+			if err != nil {
+				return fmt.Errorf("failed to generate verifier config: %w", err)
+			}
+			jb, resp, err := clClient.CreateJobRaw(committeeVerifierSpec(string(tomlConfig)))
+			if err != nil {
+				return fmt.Errorf("failed to create committee verifier job: %w", err)
+			}
+			if resp.StatusCode != http.StatusOK && resp.StatusCode != http.StatusCreated {
+				return fmt.Errorf("failed to create committee verifier job: %s", resp.Status)
+			}
+			Plog.Info().
+				Int("CurrentIndex", index).
+				Any("CommitteeVerifierJob", jb).
+				Msg("Created committee verifier job on node")
+		case services.Standalone:
+			continue
+		}
+	}
+
+	if executorIn != nil && executorIn.Mode == services.CL {
```

Comment:
Right now we only have one executor but we should probably have several to match a more realistic setup.